config.go

```diff
@@ -15,9 +15,5 @@
 package mysql
 
 type Config struct {
-	Host     string `json:"host" validate:"required"`
-	Port     int    `json:"port" default:"3306"`
-	User     string `json:"user" validate:"required"`
-	Password string `json:"password" validate:"required"`
-	Database string `json:"database" validate:"required"`
+	URL string `json:"url" validate:"required"`
```

Comment:
Let's add a comment above this field to describe to the user what it does. To give you more insight - paramgen extracts the comment into the parameter description which is included in the specifications and then reaches the end user through the API. It's displayed in the UI, in the future we will also extract it into the readme so you won't have to manually repeat it there.

Reply:
I used to have a very scrappy gpt 4 tool that made me the markdown table, but that's not really usable for automating stuff (at least without any api keys).